safe-defaults/tmpfiles-disable.conf.in

```diff
@@ -1,4 +1,5 @@
-r @SYSCONFDIR@/NetworkManager/conf.d/opendns-family-shield.conf
```

Comment:
For completeness we may also want to keep this `r` for the old file.

Reply:
ok